nemo/collections/nlp/modules/common/megatron/fused_softmax.py

```diff
@@ -24,6 +23,7 @@
 
 except (ImportError, ModuleNotFoundError):
     HAVE_APEX = False
+    from nemo.collections.nlp.modules.common.megatron.enums import AttnMaskType, LayerType, ModelType
```

Comment:
What about the symbol for FusedScaleMaskSoftmax?

Reply:
about what?